### DISCUSSION TITLE
OS Zoomstack

### DISCUSSION THREAD

**Discussion by neilromig** [andreynovikov/trekarta#165] [Q&A]
For the UK, Ordnance Survey have released Zoomstack - available as a 2.75GB MBTiles map. Is there a way I can replace the OpenStreetMap tiles with this file, or be able to switch from one map to the other?

**Comment by andreynovikov** [comment]
Is it raster or vector? Where can I find it?

**Reply by neilromig** [reply]
Its at https://osdatahub.os.uk/downloads/open/OpenZoomstack but might only be available from GB.
